index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portofolio - Here we go</title>
+  </head>
+  <body>
+    <div id="app">
+      <header class="header">
+        <a href="#"><h2>My Logo</h2></a>
+        <a href="#"
+          ><img src="/img/icons/Union.png" alt="toggle menu icon"
+        /></a>
```

Comment:
- [ ] Great job with how the mobile menu looks, but since this is a menu (or so far the representation of one), it would be more semantic if you could please wrap your anchor inside of a `<nav><\nav>` tag while still leaving the logo outside only wrapped by the header

Reply:
Cool.
What do you mean about the logo?

Reply:
I mean that the logo should not be inside the newly added nav tag, it should only be inside the header, as this is a common mistake students make when we tell them about having the nav wrapping the menu

Reply:
Oh!!! Great, I get it.